src/aui-layout/js/aui-layout-builder-resize-col.js

```diff
@@ -280,26 +280,6 @@ A.LayoutBuilderResizeCol.prototype = {
     },
 
     /**
-     * Checks if a drag node can be dragged to at least one valid position.
-     *
-     * @method _canDrag
-     * @param {Node} dragNode
-     * @protected
-     */
-    _canDrag: function(dragNode) {
-        var index;
-
-        for (index = 0; index < BREAKPOINTS.length; index++) {
-            if (BREAKPOINTS[index] !== dragNode.getData('layout-position') &&
-                this._canDrop(dragNode, BREAKPOINTS[index])) {
```

Comment:
@thiago-rocha. please, double check the necessity of this method. I couldn't understand the logic inside of the method _canDrop to evaluate this condition.